### PR TITLE
fix(mat): qr scanner amount

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-653](https://emurgo.atlassian.net/browse/YOMO-653)

[YOMO-653]: https://emurgo.atlassian.net/browse/YOMO-653?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/yoroi-wallets/types/other.ts [hunk 1]
```diff
@@ -282,6 +282,7 @@ export type TxStatusResponse = {
 // Pricing api
 export const supportedCurrencies = Object.freeze({
   ADA: 'ADA',
+  TADA: 'TADA',
```
Comment (banklesss) [inline apps/wallet-mobile/src/yoroi-wallets/types/other.ts]: I have doubts about this, what do you think?

File: apps/wallet-mobile/src/yoroi-wallets/types/other.ts [hunk 2]
```diff
@@ -298,6 +299,10 @@ export const configCurrencies = {
     decimals: 6,
     nativeName: 'Cardano',
   },
+  [supportedCurrencies.TADA]: {
+    decimals: 6,
+    nativeName: 'Cardano',
+  },
```
Comment (banklesss) [inline apps/wallet-mobile/src/yoroi-wallets/types/other.ts]: I have doubts about this, what do you think?

File: apps/wallet-mobile/src/yoroi-wallets/types/other.ts (hunk 1 above)
Comment (banklesss) [reply apps/wallet-mobile/src/yoroi-wallets/types/other.ts]: removed

File: apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/InputReceiver/ReadQRCodeScreen.tsx [hunk 3]
```diff
@@ -26,7 +26,7 @@ export const ReadQRCodeScreen = () => {
           receiverChanged(address ?? '')
           const amount = pastedFormatter(params?.amount ?? '')
           tokenSelectedChanged(wallet.primaryTokenInfo.id)
-          amountChanged(amount as Quantity)
+          amountChanged(Quantities.integer(asQuantity(amount), configCurrencies[supportedCurrencies.ADA].decimals))
```
Comment (banklesss) [inline apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/InputReceiver/ReadQRCodeScreen.tsx]: at the moment only ADA (or TADA in testnet) is used

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/InputReceiver/ReadQRCodeScreen.tsx (hunk 3 above)
Comment (banklesss) [reply apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/InputReceiver/ReadQRCodeScreen.tsx]: refactored

**Pull request merged**